h2o-docs/src/product/data-science/algo-params/distribution.rst

```diff
@@ -80,6 +80,87 @@ Some examples of response distributions are provided below.
+|                   | - :math:`-2w*(y*f-exp(f))`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Gamma             | - With :math:`f=link(f)`                                                                  |
+|                   | - :math:`2w*(y*exp(-f)+f)`                                                                |
++-------------------+-------------------------------------------------------------------------------------------+
+| Laplace           | :math:`w*\mid{y-f}\mid`                                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Quantile          | - :math:`y>f: w*QuantileAlpha*(y-f)`                                                      |
+|                   | - :math:`y<=f: w*1-QuantileAlpha*(f-y)`                                                   |
++-------------------+-------------------------------------------------------------------------------------------+
+| Huber             | - :math:`\mid{y-f}\mid <= \text{Huber_Delta}`:                                            |
+|                   | - :math:`w*(y-f)^2` (Equivalent to wMSE)                                                  |
+|                   | - :math:`y<=f`:                                                                           |
+|                   | - :math:`2w*(\mid{y-f}\mid - \text{Huber_Delta})*\text{Huber_Delta}` (Equivalent to wMAE) |
++-------------------+-------------------------------------------------------------------------------------------+
+| Modified_Huber    | - with :math:`yf=(2y-1)*f`                                                                |
+|                   | - :math:`yf<-1`:                                                                          |
+|                   | - :math:`-4w*(yf)`                                                                        |
+|                   | - :math:`yf>1` : :math:`0`                                                                |
+|                   | - else: :math:`w*(yf)^2`                                                                  |
++-------------------+-------------------------------------------------------------------------------------------+
+| Tweedie           | with :math:`f=\text{link}(f)`, requires :math:`1>\text{Tweedie_Power}<2`:                 |
+|                   | :math:`2w*y^{(2-\text{Tweedie_Power})/((1-\text{Tweedie_Power})*(2-\text{Tweedie_Power})) |
+|                   | -y*\text{exp}(f*(1-\text{Tweedie_Power}))/(1-\text{Tweedie_Power})                        |
+|                   | +\text{exp}(f*(2-\text{Tweedie_Power}))/(2-\text{Tweedie_Power})}`                        |
++-------------------+-------------------------------------------------------------------------------------------+
```

Comment:
Tweedie formating looks off when I view it here: https://github.com/h2oai/h2o-3/blob/5f6197eba3a69b0fc0a0e142272e7bcbb48ec331/h2o-docs/src/product/data-science/algo-params/distribution.rst
![image](https://user-images.githubusercontent.com/57646425/129623608-c8a5b3ae-1971-46f0-823c-d1390cd0b099.png)

Not sure if this going to be fixed.

Reply:
Here is how the Tweedie formatting looks when the page is built. Does it still look off?

https://user-images.githubusercontent.com/52463461/129743156-456893d1-01bc-4367-94c7-570ea18b1a7f.mov

Reply:
For first part, It should be "requires 1> _tweediePower > 2 ", not "requires 1> _tweediePower < 2" 
The .mov shows everything in the exponent. But I think it should have three parts: 2w*y^(blah) -y*exp(blah)+exp(blah). For lines below, first line needs to end with } and remove last } on third line. Like this: 
`2w*y^{(2-\text{Tweedie_Power})/((1-\text{Tweedie_Power})*(2-\text{Tweedie_Power}))} |
|                   | -y*\text{exp}(f*(1-\text{Tweedie_Power}))/(1-\text{Tweedie_Power})                        |
|                   | +\text{exp}(f*(2-\text{Tweedie_Power}))/(2-\text{Tweedie_Power})`

Reply:
With the update:

https://user-images.githubusercontent.com/52463461/129941718-38d2a9bb-6951-4c39-b68b-c3ddb213d69d.mov